doc/version_history.rst

```diff
@@ -6,7 +6,11 @@
 Version History
 ===============
 
-.. towncrier release notes start
+=======
```

Comment:
Please, make a news fragment instead

Reply:
Sorry, I don't understand what you mean here.

Reply:
See: https://github.com/lsst-ts/ts_externalscripts/blob/develop/doc/news/README.rst